src/tiled/commandbutton.cpp

```diff
@@ -34,6 +35,7 @@ using namespace Tiled::Internal;
 CommandButton::CommandButton(QWidget *parent)
     : QToolButton(parent)
     , mMenu(new QMenu(this))
+    , mCommandManager(new CommandManager(this))
```

Comment:
I created this instance so I could connect the signal `QMenu::aboutToShow` to the slot `CommandManager::populateMenu`. How can I do this without creating a new instance?